OrbitGl/PickingManager.h

```diff
@@ -101,22 +97,24 @@ class PickingManager {
   void Pick(uint32_t a_Id, int a_X, int a_Y);
   void Release();
   void Drag(int a_X, int a_Y);
-  Pickable* GetPicked();
-  Pickable* GetPickableFromId(uint32_t id);
-  bool IsDragging() const;
+  std::weak_ptr<Pickable> GetPicked();
+  std::weak_ptr<Pickable> GetPickableFromId(uint32_t id);
+  bool IsDragging();
+
+  Color GetPickableColor(std::weak_ptr<Pickable> pickable,
+                         PickingID::BatcherId batcher_id);
 
-  Color GetPickableColor(Pickable* pickable, PickingID::BatcherId batcher_id);
+  bool IsThisElementPicked(Pickable* pickable);
```

Comment:
If GetPicked got `const` this could also be const. :-)

Reply:
I can even add two consts here! ;-)